app/views/staff/sessions/new.html.erb

```diff
@@ -1,18 +1,24 @@
 <h1 class="govuk-heading-l">Log in</h1>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <%= f.govuk_error_summary %>
+<% unless FeatureFlags::FeatureFlag.active?(:enable_login_with_azure) %>
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <%= f.govuk_error_summary %>
 
-  <%= f.govuk_email_field :email, autocomplete: "email", label: { text: "Email" } %>
-  <%= f.govuk_password_field :password, autocomplete: "current-password", label: { text: "Password" } %>
+    <%= f.govuk_email_field :email, autocomplete: "email", label: { text: "Email" } %>
+    <%= f.govuk_password_field :password, autocomplete: "current-password", label: { text: "Password" } %>
 
-  <% if devise_mapping.rememberable? %>
-    <%= f.govuk_check_boxes_fieldset :remember_me, multiple: false, legend: nil do %>
-      <%= f.govuk_check_box :remember_me, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Remember me?" } %>
+    <% if devise_mapping.rememberable? %>
+      <%= f.govuk_check_boxes_fieldset :remember_me, multiple: false, legend: nil do %>
+        <%= f.govuk_check_box :remember_me, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Remember me?" } %>
+      <% end %>
     <% end %>
+
+    <%= f.govuk_submit %>
   <% end %>
+<% end %>
 
-  <%= f.govuk_submit %>
+<% if FeatureFlags::FeatureFlag.active?(:enable_login_with_azure) %>
+  <%= govuk_button_to "Sign in with Active Directory", omniauth_authorize_path(:staff, :azure_activedirectory_v2), method: :post, data: { turbo: "false" } , id: "azure_ad_login"%>
```

Comment:
```suggestion
  <%= govuk_button_to "Sign in with Active Directory", omniauth_authorize_path(:staff, :azure_activedirectory_v2), method: :post, id: "azure_ad_login"%>
```

Do we need the `turbo: "false"` bit?

Reply:
I think what the turbo: false is doing is disabiling turbolinks from packaging this request as an ajax request. Im unsure if it should go.

Reply:
We don't use turbo links in this repo so I think we can get rid of it.